src/troubleshooting/shortnames.rst

```diff
@@ -0,0 +1,287 @@
+SSSD and shortnames
+###################
+
+Lets start with the definition of ``fully qualified name``:
+
+.. note::
+
+    Fully qualified user name is the situation when user name is followed with domain name,
+    for example ```bob@corp.abc.com`` instead of just ``bob``. On Windows systems it can be called
+    ``SAM`` account (``Security Account Manager``) and writted different way: ``domain_name\user_name``.
+    UPN (User Principal Name) is another alias for this naming convention.
+    In case of Active Directory (AD) this type of entry may be called ``DN``
+    (``Distinguished Name``), where separated blocks will have value:
+    CN=bob,OU=Users,DC=corp,DC=abc,DC=net
```

Comment:
Just mention the two formats: bob@domain and DOMAIN\bob (in the second case it is not really a domain name but a netbios name, which is little bit different thing - also called a flatname in sssd code base).

SAM and UPN are not relevant for description of a fully qualified name. SAMAccountName is attribute that holds username in AD. UPN is attribute that stores Kerberos principal name.